### PR TITLE
made it so Github installation instructions have consistent code tags

### PR DESCRIPTION
**Pull request by mowchan**
@jennEDVT I created a new pull request for you here.

### CODE REVIEW

File: views/guide.lavender [hunk 1]
```diff
@@ -89,8 +89,9 @@ block content
           h2#github via Github      
           p
             | Add the lavender submodule with 
-            code git submodule add git@github.com:golavender/lavender.git (folder to 
-            | clone to) and include it with 
+            code git submodule add git@github.com:golavender/lavender.git (folder to
+            code clone to)
```
Comment (mowchan) [inline views/guide.lavender]: You shouldn't use `code` on each line. Instead you can keep it basically how it was before. Just be sure to indent the second line.

```
code git submodule add git@github.com:golavender/lavender.git (folder to
  | clone to)
| and include it with 
```

Comment (sugarPhlox) [inline views/guide.lavender]: Cool tidbit here for you @jennEDVT , if it's a small change like this, you can just do a quick edit in GitHub. Go to the 'Files Changed' tab, and click on the pencil to the left of the file name (it should pop up saying something about file changes).
Do a quick fix, commit the changes, and it's done. Just don't go crazy and do lines and lines of code.

Comment (jennEDVT) [inline views/guide.lavender]: Thanks Cara! I'll definitely do that in the future. I'm gonna go through
the whole, longer process for this one again this time, (even though it's
just a teensy change) just for the sake of my own learning. I want to make
sure I can commit changes and do pull requests correctly...

**Comment by jennEDVT** [comment]
Thanks Dan!

**Comment by sugarPhlox** [comment]
:cake: 

Good job learning the github system!

**Comment by mowchan** [comment]
:cake:

**Comment by mowchan** [comment]
Hey @jennEDVT so there are some conflicts that need to be resolved before this can be merged. You can follow these steps to resolve them.

In the Terminal, make sure you're on this branch and run `git fetch` to pull down any changes on development that you may not have already.

Next, run `git merge origin/development'. The terminal will output the files which have conflicts. Now, open the file(s) in your text editor and fix them. 

To help walk you through this, I pulled down the files and tried the merge myself. This is what the conflicts will look like: 

```
<<<<<<< HEAD
            code git submodule add git@github.com:golavender/lavender.git (folder to
              |clone to)
            | and include it with 
            code require '(where you put lavender)/src/Lavender/lavender.php'   
=======
            code.language-markup git submodule add git@github.com:golavender/lavender.git (folder to 
            | clone to) and include it with 
            code.language-markup require '(where you put lavender)/src/Lavender/lavender.php'   
>>>>>>> origin/development
```

To fix, you need to delete the code that is no longer applicable and remove the `<<<<<<< HEAD`, `=======`, and `>>>>>>> origin/development`. Do note that the class `.language-markup` needs to be on your `code`.

Once you have your changes, go back to the Terminal and run `git commit`. An automatically generated commit message will come up, simply type `:wq` and press enter.

`git push` and you're all set. It should now be able to merge into development.

**Comment by jennEDVT** [comment]
Dan, thanks for taking the time to walk me through that. I've done most of
the necessary steps, however I have a sort of dumb question that I'm stuck
on. I've done everything up through 'git commit' and got the automatically
generated commit message shown in the screenshot below. I believe my next
step is to type  ':wq' and press enter, but when I try that nothing happens
(pressing enter simply takes me to a new line.) Where is that I should be
typing :wq?

[image: Inline image 1]

UPDATE: Nevermind, Dan. I just quit out of that editor and then did git push. I looks like that all worked fine...

Thanks!

Jenn

**Pull request merged**